services/stt/local/Makefile

```
@@ -3,18 +3,34 @@ PROTO_OUT = .
 
 PROTOS := $(shell find $(PROTO_DIR) -name '*.proto')
 
-.PHONY: proto setup test test-unit test-integration clean-proto format
+.PHONY: proto setup test test-unit test-integration clean-proto rebuild-proto format proto-check
```

checkmake (0.2.2)

[warning] Missing required phony target "all"

(minphony)